worker/docker.go

```diff
@@ -309,10 +315,11 @@ func removeExistingContainers(ctx context.Context, client *client.Client) error
 	return nil
 }
 
+// dockerContainerName generates a unique container name based on the pipeline, model ID, and a random UUID.
 func dockerContainerName(pipeline string, modelID string) string {
-	// text-to-image, stabilityai/sd-turbo -> text-to-image_stabilityai_sd-turbo
-	// image-to-video, stabilityai/stable-video-diffusion-img2vid-xt -> image-to-video_stabilityai_stable-video-diffusion-img2vid-xt
-	return strings.ReplaceAll(pipeline+"_"+modelID, "/", "_")
+	uuid := uuid.New().String()
```

Comment:
@ad-astra-video changed with uuid and improved the format a bit -> `text-to-image_SG161222-RealVisXL-V4.0-Lightning_8687d044-771f-4dca-9aee-bb56390d2ad6`. This makes the distinction between the three variables more clear.